.github/workflows/gcc-problems.yml

```diff
@@ -121,15 +139,19 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: build_logs_oai
-          path: ${{ github.workspace }}/compile.log      
+          path: ${{ github.workspace }}/compile.log
 
   build_session_manager:
+    needs:
+      - gen_build_container
```

Comment:
put path filter first as this will be executed first

Reply:
@quentinDERORY - Addressed the comments and pushed the changes